packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2319,7 +2322,8 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         return snapshotRefSeq;
     }
 
-    private async fetchSnapshotFromStorage(versionId: string, logger: ITelemetryLogger, event: ITelemetryGenericEvent) {
+    private async fetchSnapshotFromStorage(
+        versionId: string | null, logger: ITelemetryLogger, event: ITelemetryGenericEvent) {
```

Comment:
Are you sure null will actually work here and do the right thing for all services? Glancing at the code in ODSP, I believe it will, but please test your change with R11s stack and ODSP stack (using any example)

Reply:
it is working with both servers when i ran clicker example. i will run e2e test to check as well